src/components/Filter.tsx

```diff
@@ -1,34 +1,32 @@
-import React, { ReactElement } from 'react';
-import { Form, FormGroup, Label, Input } from 'reactstrap';
+import React from 'react';
+import Select from 'react-select';
+import _ from 'lodash';
 
 const Filters = ({ layers, setLayers }) => {
-  const y = [];
-  Object.keys(layers).map(function (key, index) {
-    y.push(
-      <FormGroup check>
-        <Label check>
-          <Input
-            type="checkbox"
-            checked={layers[key]}
-            name={key}
-            onChange={handleLayer}
-          />{' '}
-          {key}
-        </Label>
-      </FormGroup>,
-    );
-  });
+  const selectOptions = _.map(layers, (layer, key) => ({
+    value: key,
+    label: key,
+  }));
 
-  function handleLayer(event: { target: { name: string; checked: boolean } }) {
-    const keyx = event.target.name;
-    const newLayers = { ...layers };
-    newLayers[keyx] = event.target.checked;
+  function handleLayer(event: any) {
+    const selectedLayers = _.map(event, (layer) => layer.label);
+    const newLayers: { [x: string]: boolean } = {};
+    _.forEach(selectOptions, (option) => {
```

Comment:
should we be using lodash commands here if the default js arrays support them? If we aren't using any special lodash commands, we could remove it entirely and reduce our bundle size!